views/lib/topbar.hbs

```diff
@@ -0,0 +1,60 @@
+<nav id="top-navbar" class="topbar">
+  <button class="icon-btn sidebar-toggle" title="{{$t "lib.loggedin.label.sidebarMenu"}}" aria-label="{{$t "lib.loggedin.label.sidebarMenu"}}">
```

Comment:
Looks like the toggle button keeps a blue focus border. Is that intended?
![image](https://github.com/hpi-schul-cloud/schulcloud-client/assets/72452466/2d3f8f5c-2caa-406c-bf10-5449eb4a9230)

Reply:
At least it's the common behavior of the buttons in the legacy client apparantly